### PR TITLE
DTAv2 - Test platform 2017 support

### PR DESCRIPTION
**Pull request by nigurr**
PR for master: https://github.com/Microsoft/vsts-tasks/pull/3301

### CODE REVIEW

File: make-util.js [hunk 1]
```diff
@@ -536,6 +536,22 @@ var getExternals = function (externals, destRoot) {
             copyGroups(package.cp, packageSource, destRoot);
         });
     }
+
+    // for any file type that has to be shipped with task
```
Comment (acesiddhu) [inline make-util.js]: do we need this change in m111

File: Tasks/DeployVisualStudioTestAgent/task.loc.json [hunk 2]
```diff
@@ -36,20 +38,20 @@
...
-      "name": "testMachineGroup",
+      "name": "testMachines",
       "type": "multiLine",
-      "label": "ms-resource:loc.input.label.testMachineGroup",
+      "label": "ms-resource:loc.input.label.testMachines",
       "defaultValue": "",
       "required": true,
-      "helpMarkDown": "ms-resource:loc.input.help.testMachineGroup",
+      "helpMarkDown": "ms-resource:loc.input.help.testMachines",
       "groupName": "testMachineGroups"
```
Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/task.loc.json]: we shuld rename thegrp name also

File: make-util.js (hunk 1 above)
Comment (nigurr) [inline make-util.js]: yes

File: Tasks/DeployVisualStudioTestAgent/task.loc.json (hunk 2 above)
Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/task.loc.json]: I kept *group because testMachine is group Name. not because of `testMachineGroup` ;)

**Pull request closed without merging**